Assignment2/lesmesrafa-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,55 @@
+# JSON
+
+{
+	
+	"fullname":"Rafael Martin Lesmes",
+	"hometown":"Boadilla del Monte",
+	"age":"21",
+	"almamater":"Universidad Politecnica de Madrid",
+	"parentsname":["Ismael Martin Barroso","Maria Magdalena Lesmes Arenas"],
+	"subjects":{
+		"Semantic Web, Linked Data and Knowledge Graphs":[{"teachers":["Oscar Corcho", "Paola Espinoza", "Raul Garcia Castro"],
+		"description":"The objective of this course is to gain theoretical and practical knowledge about the methodological and technological foundations of the generation and publication  of vocabularies and data on the Web of Linked Data, as well as knowledge graphs."}],
+
+		"Nanotecnologia":[{"teachers":["Maria Estibaliz Martinez Izquierdo","Maria Mercedes Perez Castellanos"],
+		"description":"En esta asignatura, se pretende dar al alumno una formación introductoria sobre las diferentes investigaciones y aplicaciones que existen actualmente dentro de la Nanotecnología, en los diferentes campos científicos que abarca"}]
+
+	}
+}
+
+
+# JSON-LD
+{
+
+	"@context":{
+		"root":"http://www.example.es/api/example",
+      	"rdfs":"http://www.w3.org/2000/01/rdf-schema#",
+      	"vcard":"http//www.w3.org/2006/vcard/ns#",
+      	"age":"rdfs:label",
+      	"teachers":"rdfs:label",
+      	"description":"rdfs:label"
+      

```

Comment:
You also have to define properties like hometown, age, almamater,etc. in the @context as it will allow you to define what specific vocabulary property you have to use to annotate your data.

Reply:
I have already checked it again